front/room_list.html

```diff
@@ -0,0 +1,65 @@
+<html>
+    <head>
+        <script src="https://unpkg.com/vue@next"></script>
+        <link rel="stylesheet" href="css/global.css">
+        <link rel="stylesheet" href="css/room_list.css">
+    </head>
+    <body>
+        <div id="app">
+            <room-list v-model:rooms="rooms" v-on:room-selected="displayRoom($event)"></room-list>
+        </div>
+        <script>
+            rooms = [
+                {
+                    "_id": {
+                        "$oid": "6071ac5176e5bd456a859d9d"
+                    },
+                    "description": "ma description",
+                    "title": "Room 1",
+                    "user": []
+                },
+                {
+                    "_id": {
+                        "$oid": "6071ac5176e5bd456a859d9d"
+                    },
+                    "description": "ma description",
+                    "title": "Room 2",
+                    "user": []
+                },
+                {
+                    "_id": {
+                        "$oid": "6071ac5176e5bd456a859d9d"
+                    },
+                    "description": "ma description",
+                    "title": "Room 3",
+                    "user": []
+                }
+            ];
+            const roomList = {
+                name: "room-list",
+                props: ["rooms"],
+                emits: ["room-selected"],
+                template: `
+                <div class='room_list_wrapper'>
+                    <div class="room_list_item" v-for="room in rooms" @click="$emit('room-selected', room)">
```

Comment:
Add [key](https://v3.vuejs.org/guide/migration/key-attribute.html#with-template-v-for) to help vueJS update the list : `:key='room._id.$oid'`